### PR TITLE
Fixes for production errors

### PR DESCRIPTION
**Pull request by ChrisBr**

### CODE REVIEW

**Comment by hennevogel** [comment]
Please fix the rubocop offenses :-)

File: app/models/comment.rb [hunk 1]
```diff
@@ -41,7 +41,11 @@ def has_children?
   }
 
   scope :find_since_last_login, lambda { |user|
-    where(created_at: (user.last_sign_in_at..Time.now)).order(created_at: :desc)
+    if user.last_sign_in_at
+      where(created_at: (user.last_sign_in_at..Time.now)).order(created_at: :desc)
+    else
+      none
```
Comment (hennevogel) [inline app/models/comment.rb]: none?

Comment (ChrisBr) [inline app/models/comment.rb]: http://apidock.com/rails/v4.0.2/ActiveRecord/QueryMethods/none

**Comment by ChrisBr** [comment]
@hennevogel fixed!

File: app/models/user.rb [hunk 2]
```diff
@@ -74,7 +74,9 @@ def self.for_ichain_username(username, attributes)
     raise UserDisabled if user && user.is_disabled
 
     if user
-      user.update_attributes(email: attributes[:email])
+      user.update_attributes(email: attributes[:email],
+                             last_sign_in_at: user.current_sign_in_at,
+                             current_sign_in_at: Time.current)
```
Comment (hennevogel) [inline app/models/user.rb]: Isn't that happening in devise somewhere?

File: app/models/comment.rb (hunk 1 above)
Comment (hennevogel) [inline app/models/comment.rb]: You leanr something new every day... awesome!

File: app/models/user.rb (hunk 2 above)
Comment (ChrisBr) [inline app/models/user.rb]: Obviously not :(
Your last login was somewhere 2014 and I have no last login set at all ...

Comment (hennevogel) [inline app/models/user.rb]: Heh awesome. Let's merge this and create an issue to find a better solution. Can you do that please? Thanks...

Comment (ChrisBr) [inline app/models/user.rb]: Done in https://github.com/openSUSE/devise_ichain_authenticatable/issues/2

**Pull request merged**